committee-steering/governance/sig-governance-template-short.md

```diff
@@ -0,0 +1,125 @@
+
+- Contributing instructions defined in the SIG CONTRIBUTING.md
+
+### Project management
+
+#### Subproject creation
+
+---
+
+Option 1: by SIG Technical Leads
+
+- Subprojects may be created by [KEP] proposal and accepted by lazy consensus with fallback on majority vote of
+  SIG Technical Leads.  The result *SHOULD* be supported by the majority of SIG members.
+  - KEP *MUST* establish subproject leads
+  - [sigs.yaml] *MUST* be updated to include subproject information and [OWNERS] files with subproject leads
+  - Where subprojects processes differ from the SIG governance, they must document how
+    - e.g. if subprojects release separately - they must document how release and planning is performed
+
+Option 2: by federation of subprojects
+
+- Subprojects may be created by [KEP] proposal and accepted by lazy consensus with fallback on majority vote of
+  subproject leads in the SIG.  The result *SHOULD* be supported by the majority of members.
+  - KEP *MUST* establish subproject leads
+  - [sigs.yaml] *MUST* be updated to include subproject information and [OWNERS] files with subproject leads
+  - Where subprojects processes differ from the SIG governance, they must document how
+    - e.g. if subprojects release separately - they must document how release and planning is performed
+
+---
+
+- Subprojects must define how releases are performed and milestones are set.  Example:
+
+> - Release milestones
+>   - Follows the kubernetes/kubernetes release milestones and schedule
+>   - Priorities for upcoming release are discussed during the SIG meeting following the preceding release and
+>     shared through a PR.  Priorities are finalized before feature freeze.
+> - Code and artifacts are published as part of the kubernetes/kubernetes release
+
+### Technical processes
+
+Subprojects of the SIG *MUST* use the following processes unless explicitly following alternatives
+they have defined.
+
+- Proposing and making decisions
+  - Proposals sent as [KEP] PRs and published to googlegroup as announcement
+  - Follow [KEP] decision making process
+
+- Test health
+  - Canonical health of code published to <link to dashboard>
+  - Consistently broken tests automatically send an alert to <link to google group>
+  - SIG members are responsible for responding to broken tests alert.  PRs that break tests should be rolled back
+    if not fixed within 24 hours (business hours).
+  - Test dashboard checked and reviewed at start of each SIG meeting.  Owners assigned for any broken tests.
+    and followed up during the next SIG meeting.
+
+Issues impacting multiple subprojects in the SIG should be resolved by either:
+
+- Option 1: SIG Technical Leads
+- Option 2: Federation of Subproject Leads
+
+[lazy-consensus]: http://communitymgt.wikia.com/wiki/Lazy_consensus
+[super-majority]: https://en.wikipedia.org/wiki/Supermajority#Two-thirds_vote
+[warnocks-dilemma]: http://communitymgt.wikia.com/wiki/Warnock%27s_Dilemma
```

Comment:
These links look copied from the long template, with some not used and probably should be (eg: lazy consensus) and some not used and probably need removed (eg: warnocks dilemma and slo).

Reply:
Done